project.clj

```diff
@@ -42,7 +42,7 @@
                  ;; Schemas
                  [prismatic/schema ~schema-version]
                  [metosin/schema-tools ~schema-tools-version]
-                 [threatgrid/ctim "0.4.21"]
+                 [threatgrid/ctim "0.4.22-SNAPSHOT"]
```

Comment:
I thought that `Investigation` was not supposed to make it on CTIM  and only defined on CTIA?

Reply:
We will need to use it on the UI too, so I think CTIM makes sense.

Reply:
Though it would be easy to move to CTIA if that's what we really want. I'm not sure how "pure" CTIM really needs to be.  I'll go out on a limb and say I think it's not important.